server/cmwell-cons/src/main/scala/ComponentConf.scala

```diff
@@ -646,7 +646,7 @@ case class CwConf(home: String,
           "-Dcom.sun.management.jmxremote.ssl=false",
           "-Dcom.sun.management.jmxremote.authenticate=false",
           "-Duser.timezone=GMT0",
-          s"-Dcrashableworker.subjectsAreHttps=$subjectsInSpAreHttps"
+          s"-Dcrashableworker.subjectsAreHttps=${subjectsInSpAreHttps.mkString(",")}"
```

Comment:
remove this line. no -Ds please....

Use HOCON and create arrays and then `config.getStringList`
In order to programmatically configure it use the `{{}}` of mkConf